vyper/codegen/expr.py

```diff
@@ -800,74 +791,70 @@ def build_in_comparator(self):
         left = Expr(self.expr.left, self.context).lll_node
         right = Expr(self.expr.right, self.context).lll_node
 
-        result_placeholder = self.context.new_internal_variable(BaseType("bool"))
-        setter = []
+        # temporary kludge to block #2637 bug
```

Comment:
Is this a TODO?

Reply:
added, https://github.com/vyperlang/vyper/pull/2633/commits/8c0ae85d8ecfbe2a7572e999e7188179fdf37362